doc/sphinx/practical-tools/utilities.rst

```diff
@@ -453,50 +491,6 @@ line timing data:
       This target requires python to build the table.
 
 
-Reusing/extending the generated Makefile
-++++++++++++++++++++++++++++++++++++++++
-
-Including the generated makefile with an include directive is
```

Comment:
I want to keep this sentence, please don't remove it.

Reply:
Could this sentence be embedded in a previous sentence, as in:
```
the advantage of a wrapper (compared e.g. to including the generated makefile with an include directive) is that it provides a way independent of the version of Coq to regenerate a ``Makefile``
```
?

Reply:
One drawback is that you can't have your own target for generating the included Makefile, because the generated Makefile includes a target regenerating itself, and there's not a good way to override that target.

Reply:
So:
```
... the advantage of a wrapper (compared to directly calling the generated makefile
or to including the generated makefile with an include directive) is that it provides a way 
independent of the version of Coq to regenerate a ``Makefile`` ...
```
?

Reply:
Saying that A has advantages over B is not the same thing as saying that you should no do B.
The reasons are other, like that `inclusion` is not modular, all variables and rules are injected into the same name space.
This is why there is Makefile.conf that only include variables assignment and all variables are named COQMF_*, that one you can include.